sdk/include/platform/sunburst/platform-i2c.hh

```diff
@@ -233,6 +233,9 @@ struct OpenTitanI2c
 	  FifoControlTransmitReset = 1 << 8,
 	};
 
+	static constexpr uint32_t RXFIFODepth = 8;
+	static constexpr uint32_t FmtFIFODepth = 8;
```

Comment:
```suggestion
	static constexpr uint32_t ReceiveFifoDepth = 8;
	static constexpr uint32_t FormatFifoDepth = 8;
```

Reply:
I would like to leave this as `RXFIFODepth` as it's exclusively referred to as the 'RX FIFO' in the documentation

Reply:
This is an CHERIoT RTOS style decision

Reply:
Well I guess we can see what happens when we upstream it.

Ultimately the object being referred to is the 'RX FIFO' not the 'Receive FIFO' in the documentation and RTL so it seems better to stay in step with it.

Reply:
It would be nice to keep with the CHERIoT RTOS style decision. You can always put a comment with "RX FIFO" so that if people are searching they can easily find it.

Reply:
Though that said a similar thing has happened to the register names such that they don't precisely match the documentation so looks like a battle I'll loose, I'll revert to 'ReceiveFifoDepth'.